devops/dockerfiles/node/Dockerfile

```diff
@@ -3,23 +3,36 @@ LABEL description="Compiles all workspace artifacts"
 WORKDIR /mangata
 COPY . /mangata
 
+#Export memory Profile env. variables.
+RUN export MEMORY_PROFILER_LOG=info && export MEMORY_PROFILER_LOGFILE=profiling_%e_%t.log && export MEMORY_PROFILER_OUTPUT=profiling_%e_%t.dat && export MEMORY_PROFILER_CULL_TEMPORARY_ALLOCATIONS=1
+
 ENV ETH_APP_ID=0xdd514baa317bf095ddba2c0a847765feb389c6a0
 ENV ERC20_APP_ID=0x00e392c04743359e39f00cd268a5390d27ef6b44
 
-RUN apt-get update && apt-get install -y cmake pkg-config libssl-dev git build-essential clang libclang-dev curl llvm
+RUN apt-get update && apt-get install -y cmake pkg-config libssl-dev git build-essential clang libclang-dev curl llvm nodejs npm
+RUN npm install --global yarn
 
 RUN rustup install nightly-2021-10-19
 RUN rustup default nightly-2021-10-19
 RUN rustup target add wasm32-unknown-unknown --toolchain nightly-2021-10-19
+#Get Mem profiler binaries
+
 RUN cargo install cargo2junit
 
 FROM env_builder as builder
+RUN mkdir memory_profiler && cd memory_profiler && git clone https://github.com/koute/memory-profiler && cd memory-profiler && git checkout  0.8.0  &&  cargo build --release -p bytehound-preload && cargo build --release -p bytehound-cli
```

Comment:
maybe it works but actully i'm not happy how its done;/ we still build profiler binaries on image built on top of 
`liuchong/rustup:stable` where we run these in another docker image  ;/ Ideally we should use just debian image and install rustup & cargo manually. But if it works (and it seems it does) we can just create ticker for later